client/starwhale/core/model/model.py

```diff
@@ -580,6 +580,9 @@ def list(
             else:
                 _info = load_yaml(_mpath)
 
+            if not _info:
```

Comment:
Is there a case that digestFile and manifestFile all of empty in a model package? Is it in an error?